about/policy/sgid/index.md

```diff
@@ -1,116 +1,171 @@
+- Datasets must define the following metadata items in the ArcGIS Online item:
+  - The **title** should include "Utah" if the rest of the title is not immediately and uniquely identifiable as a Utah dataset.
+  - The **description** must include a summary of the dataset. The first line should include the date the dataset was last updated.
+  - The **tags** should include the source entity's name and any other relevant words not found in the title (please refer to Esri's [post about using tags effectively](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/using-tags-effectively/) for more information).
+- Datasets should enable the "Allow others to export data to different formats" option in their ArcGIS Online settings to enable end users to **download** the dataset as a file geodatabase.
 
-We will populate the [minimum metadata content]({% link about/policy/metadata/index.md %}) in the SGID geodatabase. We recommended using the [metadata form](https://docs.google.com/forms/d/1u7gwdmRN-83Kh5zizi-kHRObeoNyaakM3scPkLT3zKY/edit) to create new metadata for the SGID.
+We strongly encourage dataset owners to follow these ArcGIS Online-specific guidelines, even though Open Data does not use them directly:
 
-The domain values for any new coded value domain must match their corresponding descriptions. Read about this effort in the [improving data clarity with domains](https://docs.google.com/document/d/12bdtmtv-ZVF9g-lFshbobx8DbKX9LfbL44HawLjFTVw/edit?usp=sharing) document.
+- The items' **thumbnails** appear when a user searches for the dataset on ArcGIS Online or in ArcGIS Pro. A [branded thumbnail](https://www.esri.com/arcgis-blog/products/arcgis-online/data-management/put-your-best-thumbnail-forward/) tells users that this is an authoritative dataset. For example, we use thumbnails that include the category logo for all the SGID data we share in ArcGIS Online.
+- Mark the item as **authoritative** in ArcGIS Online. This helps users decide which dataset to use when their search in ArcGIS Online returns several similar results.
 
-We will use [Sweeper](https://github.com/agrc/sweeper) (or some other form of automation) to remove all empty or invalid geometries and duplicate records.
+### Open SGID Qualifications
+{: .text-left}
 
-The SGID uses the ESRI default Resolution and Tolerance:
+- Datasets should be useful to **multiple agencies or organizations'** daily activities or decision-making. Datasets primarily used internally by a single agency aren't great candidates for the Open SGID.
+- The entity providing the dataset should not have the internal **capability** to publicly serve the data (such as through an existing ArcGIS Online organization).
```

Comment:
i'm not sure about this line: 
"The entity providing the dataset should not have the internal capability to publicly serve the data"

because we bring data into the open sgid that other agencies are hosting on their own.